iphone/Maps/Classes/CustomViews/Login/MWMAuthorizationOSMLoginViewController.mm

```diff
@@ -140,7 +140,7 @@ - (IBAction)login
       catch (exception const & ex)
       {
         LOG(LWARNING, ("Error login", ex.what()));
-        [Statistics logEvent:@"Editor_Reg_request_result" withParameters:@{kStatIsSuccess : kStatNo,
+        [Statistics logEvent:@"Editor_Auth_request_result" withParameters:@{kStatIsSuccess : kStatNo,
```

Comment:
Так ты его уже логаешь. Я только конец добавил.

> On Mar 28, 2016, at 12:47, Vlad Mihaylenko <notifications@github.com> wrote:
> 
> In iphone/Maps/Classes/CustomViews/Login/MWMAuthorizationOSMLoginViewController.mm <https://github.com/mapsme/omim/pull/2555#discussion_r57558403>:
> 
> > @@ -140,7 +140,7 @@ - (IBAction)login
> >        catch (exception const & ex)
> >        {
> >          LOG(LWARNING, ("Error login", ex.what()));
> > -        [Statistics logEvent:@"Editor_Reg_request_result" withParameters:@{kStatIsSuccess : kStatNo,
> > +        [Statistics logEvent:@"Editor_Auth_request_result" withParameters:@{kStatIsSuccess : kStatNo,
> в таком случае, нам действительно нужно логать начало?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2555/files/25fe657b1bb45c4c86dd76b93421d7c4d5c1b34d#r57558403>